python/src/od_lib/06_contributions/01_extract_contributions.py

```diff
@@ -12,51 +12,42 @@
 SPEECH_CONTENT_OUTPUT = path_definitions.SPEECH_CONTENT_STAGE_04
 CONTRIBUTIONS_EXTENDED_OUTPUT = path_definitions.CONTRIBUTIONS_EXTENDED_STAGE_01
 CONTRIBUTIONS_SIMPLIFIED = path_definitions.CONTRIBUTIONS_SIMPLIFIED
+CONTRIBUTIONS_SIMPLIFIED.mkdir(parents=True, exist_ok=True)
 
 speech_id = 0
 
-if not os.path.exists(CONTRIBUTIONS_SIMPLIFIED):
-    os.makedirs(CONTRIBUTIONS_SIMPLIFIED)
-
 contributions_simplified = pd.DataFrame(
     {"text_position": [], "content": [], "speech_id": []}
 )
+simplified_list = [contributions_simplified]
```

Comment:
```suggestion
simplified_list = []
```

this should no longer be needed, no?